### PR TITLE
[31699] Feature: Scroll mobile header with content

### PR DESCRIPTION
**Pull request by ingamai**
See ticket: https://community.openproject.com/projects/openproject/work_packages/31699

### CODE REVIEW

**Review by HDinger — COMMENTED** [review]
As discussed, there is some flickering effect when testing it on a real mobile device. On desktop it looks fine. Please do not forget about some throttling here, too.

File: app/assets/stylesheets/layout/_base.sass [hunk 1]
```diff
@@ -31,7 +31,7 @@
   @include default-transition
   display: grid
   grid-template-rows: auto 1fr
-
+  height: 100%
```
Comment (HDinger) [inline app/assets/stylesheets/layout/_base.sass]: These changes causes some weird automated scrolling effects in Safari (12.1). When scrolling down, the page scrolls up automatically afterwards. Could you please check this?
Also, why is this always applied and not only for mobile?

**Review by HDinger — COMMENTED** [review]
Please see the comment below...

**Comment by lindenthal** [comment]
Hi @ingamai und @HDinger,
just tested it on [PullPreview](https://pullpreview.com/opf/openproject/sessions/c6d2f08f70d076613e209b501cd5990c47df7271) with the latested Safari on iOS.  The header does not scroll yet. Can you please check?

Thank
Niels

**Review by HDinger — APPROVED** [review]
👍

**Pull request merged**